topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -121,6 +107,22 @@ def populate_item(self, metadata_row: Dict[str, str], asset: Optional[Asset] = N
         item.add_extension(StacExtensions.linz.value)
         item.add_extension(StacExtensions.version.value)
 
+    def population_collection(self, collection: Collection) -> None:
+        collection.description = "This aerial photographic survey was digitised from {} {} negatives in the Crown collection of the Crown Aerial Film Archive."
```

Comment:
Could this be all done in `collection.update_description()`? I get that this will be updated with the `colour` and `size` later but for understanding I would suggest to do everything at the same place.